src/main/resources/application.yaml

```diff
@@ -32,18 +32,18 @@ feature-flag:
   submit-via-api: off
   send-non-partner-county-alert: off
   document-upload-feature: on
-  county-anoka: off
-  county-carver: off
-  county-clay: off
+  county-anoka: on
+  county-carver: on
+  county-clay: on
   county-cook: off
   county-dodge: off
   county-hennepin: on
   county-morrison: off
   county-olmsted: on
-  county-ottertail: off
+  county-ottertail: on
   county-sherburne: off
   county-steele: off
-  county-stlouis: off
+  county-stlouis: on
```

Comment:
I think we're supposed to turn these off before merging https://www.pivotaltracker.com/story/show/176912511/comments/222764336

Reply:
And this is why we do PRs! Thank you for that catch